eng/validation/update-channel.ps1

```diff
@@ -34,82 +34,35 @@ function Get-AzDOHeaders()
     return $headers
 }
 
-function Get-LatestBuildResult([PSObject]$repoData)
-{
-    ## Verified that this API gets completed builds, not in progress builds
-    $headers = Get-AzDOHeaders
```

Comment:
I think same thing with Get-AzDOHeaders and Get-AzDO-Build. Looks like this was their only usage? 
potentially other functions in this file are in the same boat. . Let's only keep code that is actually being used.

Reply:
I was following release/5.0 and just realized that has zero references. I will remove that from both release/6.0 and release/5.0